sycl/test/Unit/lit.cfg.py

```diff
@@ -45,6 +45,10 @@
     # Disable device library fallback for unit tests by default.
     config.environment['SYCL_DEVICELIB_NO_FALLBACK'] = "1"
 
+# Unconditionally enable persistent device code caching, as it is off
+# by default, and some unit tests depend on it.
+config.environment['SYCL_CACHE_PERSISTENT'] = "1"
```

Comment:
Is it possible to enable this feature only for tests requiring it?
Enabling optional feature for all the tests might hide problems in default workflow.

Reply:
Done.